docs/source/json-api/index.rst

```diff
@@ -801,10 +810,14 @@ to :doc:`lf-value-specification`::
             },
             "signatories": ["Alice"],
             "contractId": "#1:0",
-            "templateId": "6cc82609ede6576e5092e8c89a2c7a658efe15ae1347fc38eb316f787fa132bc:Iou:Iou"
-        }
+            "templateId": "b70bbfbc77a4790f66d4840cb19f657dd20848f5e2f64e39ad404a6cbd98cf75:Iou:Iou"
+        },
+        "matchedQueries": [1, 2]
```

Comment:
@hurryabit Currently `"matchedQueries"` are always emitted as demonstrated in docs even if only one query is supplied; please let me know if this is undesirable.

Reply:
I think this is perfectly fine. Ignoring a field is no hard work.